pinot-common/src/main/java/org/apache/pinot/common/utils/URIUtils.java

```diff
@@ -92,7 +93,7 @@ public static String constructDownloadUrl(String baseUrl, String rawTableName, S
 
   public static String encode(String string) {
     try {
-      return URLEncoder.encode(string, "UTF-8");
+      return URLEncoder.encode(string, StandardCharsets.UTF_8);
     } catch (Exception e) {
```

Comment:
I guess we can also remove the try catch because there is no checked exception anymore. Same for other places